config/site/src/_includes/navbar.html

```diff
@@ -5,6 +5,31 @@
       <a href="{{ '/' | relative_url }}" class="text-lg font-bold hover:underline">ElasticGraph</a>
       <a href="{{ '/getting-started' | relative_url }}" class="text-base {{ site.style.link }}">Get Started</a>
 
+      <!-- Guides dropdown -->
+      <div class="relative group">
+        <button
+          class="text-base text-blue-600 dark:text-blue-400 flex items-center space-x-1">
+          <span>Guides</span>
+          {% include icons/chevron-down.svg %}
+        </button>
+        <div class="absolute left-0 pt-2 w-48 hidden group-hover:block z-50">
+          <div class="{{site.style.navbar_dropdown}}">
+            <!-- Dropdown pip -->
+            <div class="{{site.style.dropdown_pip}}"></div>
+            <div class="relative py-1">
+              <a href="{{ '/getting-started' | relative_url }}"
```

Comment:
Since this isn't under `/guides` and has a top level link, not sure it belongs here?